src/api/getFolder.js

```diff
@@ -24,7 +24,7 @@ export const getAllFolders = () => {
   return sendRequest({
     url,
     method: "GET",
-    credentials: "include",
+    credentials: false,
```

Comment:
An unintended change?

Reply:
Actually at @GMishx I was thinking that we are separately passing credentials include to every request instead we can provide it in the `sendRequest.js` file itself.